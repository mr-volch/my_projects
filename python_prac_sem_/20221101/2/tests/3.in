r = Triangle((0, 0), (20, 0), (0, 20))
s = Triangle((1, 1), (2, 2), (3, 3))
t = Triangle((2, 2), (3, 3), (4, 4))
o = Triangle((1, 1), (21, 0), (0, 21))
print(*(f"{n}({bool(x)}):{round(abs(x), 3)}" for n, x in zip("rsto", (r, s, t, o))))
print(f"{s < t=}, {o < t=}, {r < t=}, {r < s=}")
print(f"{s in t=}, {o in t=}, {r in t=}")
print(f"{r & t=}, {t & r=}, {s & r=}, {o & t=}")
