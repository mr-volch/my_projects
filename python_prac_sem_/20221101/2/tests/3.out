r(True):200.0 s(False):0.0 t(False):0.0 o(True):199.5
s < t=False, o < t=False, r < t=False, r < s=False
s in t=True, o in t=False, r in t=False
r & t=False, t & r=False, s & r=False, o & t=False
